webapp/src/shared/components/templates/ChatMessagingTemplate/ChatMessagingTemplate.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+
+import { Header, IconVariant } from '../../index';
+import { useNavigation } from '../../../hooks/UseNavigation';
+import ChatMessages, {
+  ChatMessage,
+} from './components/ChatMessages/ChatMessages';
+import ChatMessagingInput from './components/ChatMessagingInput/ChatMessagingInput';
+import { useData } from '../../../hooks/UseData';
+import socket from '../../../socket';
+import { Loading } from '../../index';
+
+import './ChatMessagingTemplate.css';
+
+export type ChatEventType = 'chatMessage' | 'chatroomMessage';
+
+type ChatMessagingProps = {
+  title: string;
+  titleNavigationUrl?: string;
+  to: string;
+  from: string;
+  blocks?: Set<string>;
+  fetchMessagesCallback: (offset: number) => Promise<ChatMessage[]>;
+  chatEvent: ChatEventType;
+  messageMapper: (msg: any) => ChatMessage;
+};
+
+export function ChatMessagingLoading() {
+  return (
+    <div className="chat-template">
+      <div className="chat-template-loading">
+        <Loading />
+      </div>
+    </div>
+  );
+}
+
+export default function ChatMessagingTemplate({
```

Comment:
praise: Thank you for adding these templates. Very nice :+1:

Reply:
Is it working for you?

Reply:
I had the same error, and generating the webapp code fixed it.

Reply:
I was in the wrong branch (vdedios/one-to-one-chat-page rather than vdedios/one-to-one-chat)

Reply:
Yes sorry, I started working on one branch and switched to other to easily solve conflicts